### PR TITLE
Create humanistica-lovaniensia-journal-of-neo-latin-studies.csl

### PR DESCRIPTION
**Pull request by POBrien333**
via https://forums.zotero.org/discussion/76329/style-request-humanistica-lovaniensia#latest

### CODE REVIEW

File: humanistica-lovaniensia-journal-of-neo-latin-studies.csl [hunk 1]
```diff
@@ -0,0 +1,298 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="note" version="1.0" page-range-format="minimal" demote-non-dropping-particle="sort-only">
+  <!-- Polyglot; journal publishes in English, French, German, Italian and Spanish. -->
+  <info>
+    <title>Humanistica Lovaniensia. Journal of Neo-Latin Studies</title>
```
Comment (adam3smith) [inline humanistica-lovaniensia-journal-of-neo-latin-studies.csl]: @rmzelle Opinion on the subtitle? The journal consistently uses it & uses it with a period. In the US (e.g. in JSTOR) it's listed with a colon. I think any of the three possible options (period, colon, no subtitle) is fine, not sure we have super clear rules on this.

Comment (rmzelle) [reply humanistica-lovaniensia-journal-of-neo-latin-studies.csl]: > I think any of the three possible options (period, colon, no subtitle) is fine, not sure we have super clear rules on this.

We never standardized on this, so I suggest to just follow whatever the journal uses.

Comment (rmzelle) [reply humanistica-lovaniensia-journal-of-neo-latin-studies.csl]: (I do prefer replacing all dashes with hyphens)

**Comment by rmzelle** [comment]
Thanks!

**Pull request merged**